### ISSUE TITLE
Understanding the logits

### ISSUE THREAD

**Issue by fremk** [UX-Decoder/Segment-Everything-Everywhere-All-At-Once#67]

**Comment by MaureenZOU** [comment]
semseg = torch.einsum("qc,qhw->chw", mask_cls, mask_pred), this step actually do a weighted sum of all the object queries, instead of selecting one object query, it actually weighted sum all the queries by the calculated weight on mask_cls
